### PR TITLE
[Bug 1143947] Community Hub: Make page size configurable.

### PR DESCRIPTION
**Pull request by mythmon**
r?

### CODE REVIEW

File: kitsune/community/api.py [hunk 1]
```diff
@@ -40,24 +52,53 @@ def get_filters(self):
...
+        return F()
+
+    def filter_page_size(self, value):
+        """Validate the page sizes, but don't return any filters."""
+        try:
+            page_size = fields.IntegerField().from_native(value)
+        except fields.ValidationError:
+            page_size = 20
+
+        if not (1 <= page_size < 100):
```
Comment (rlr) [inline kitsune/community/api.py]: nice. the first thing I thought was not having a max could be an issue

File: kitsune/community/static/less/community-new.less [hunk 2]
```diff
@@ -162,3 +158,23 @@ table.top-contributors {
...
+
+    &__selectors {
+        margin: 0;
+        flex-grow: 1;
+    }
+
+    &__pageSize {
+        flex-grow: 0;
+
+        & label {
```
Comment (rlr) [inline kitsune/community/static/less/community-new.less]: Is that `&` necessary?

File: kitsune/community/static/js/contributors-common.jsx [hunk 3]
```diff
@@ -89,7 +89,24 @@ export class Paginator extends React.Component {
...
         }
 
-        return <ol className="pagination">{pageSelectors}</ol>;
+        let page_size = this.props.filters.page_size;
+
+        return <div className="Pager">
+            <ol className="Pager__selectors pagination">{pageSelectors}</ol>
+
+            <div className="Pager__pageSize">
+                <label>Page Size</label>
```
Comment (rlr) [inline kitsune/community/static/js/contributors-common.jsx]: do we have l10n infra for these templates yet?

File: kitsune/community/static/less/community-new.less [hunk 4]
```diff
@@ -162,3 +158,23 @@ table.top-contributors {
         }
     }
 }
+
+.Pager {
+    display: flex;
+    margin-top: 20px;
+
+    &__selectors {
```
Comment (rehandalal) [inline kitsune/community/static/less/community-new.less]: oh! i like this!

File: kitsune/community/static/less/community-new.less [hunk 5]
```diff
@@ -162,3 +158,23 @@ table.top-contributors {
...
+        margin: 0;
+        flex-grow: 1;
+    }
+
+    &__pageSize {
+        flex-grow: 0;
+
+        & label {
+            margin-right: 6px;
```
Comment (rehandalal) [inline kitsune/community/static/less/community-new.less]: alphabetize

File: kitsune/community/static/less/community-new.less (hunk 4 above)
Comment (mythmon) [inline kitsune/community/static/less/community-new.less]: I know! I got really excited when this worked.

File: kitsune/community/static/less/community-new.less (hunk 2 above)
Comment (mythmon) [inline kitsune/community/static/less/community-new.less]: Nooope.

File: kitsune/community/static/js/contributors-common.jsx (hunk 3 above)
Comment (mythmon) [inline kitsune/community/static/js/contributors-common.jsx]: Not yet. We'll probably need to add that soon.

**Comment by mythmon** [comment]
:point_up: fixes.

File: kitsune/community/static/less/community-new.less (hunk 2 above)
Comment (rehandalal) [inline kitsune/community/static/less/community-new.less]: oh nice catch!

**Pull request merged**